opencti-platform/opencti-front/src/private/components/workspaces/dashboards/WidgetConfigColumnsCustomization.tsx

```diff
@@ -0,0 +1,145 @@
+                >
+                  <Checkbox
+                    checked={columns.some((col) => col.attribute === column.attribute)}
+                    onChange={() => handleToggleColumn(column.attribute)}
+                  />
+                  <ListItemText primary={formatColumnName(column)} />
+                </ListItem>
+              ))}
+            </List>
+          </Box>
+
+          {/* Selected Columns */}
+          <Box sx={{ flex: 1, height: '100%' }}>
+            <Typography variant="h4">{`${t_i18n('Selected columns')} (${columns.length})`}</Typography>
+            <DragDropContext onDragEnd={handleDragEnd}>
+              <Droppable droppableId="selected_columns">
+                {(providedDrop) => (
+                  <List
+                    ref={providedDrop.innerRef}
+                    {...providedDrop.droppableProps}
+                    sx={{
+                      border: `1px solid ${theme.palette.common.white}`,
+                      borderRadius: `${theme.borderRadius}px`,
+                      paddingBlock: theme.spacing(1),
+                    }}
+                  >
+                    {columns.map((column, index) => (
+                      <Draggable key={column.attribute} draggableId={column.attribute ?? ''} index={index}>
+                        {(providedDrag, snapshotDrag) => (
+                          <ListItem
+                            ref={providedDrag.innerRef}
+                            {...providedDrag.draggableProps}
+                            divider={index < columns.length - 1}
+                            sx={{
+                              ...providedDrag.draggableProps.style,
+                              background: snapshotDrag.isDragging ? 'rgba(0, 0, 0, 0.05)' : 'inherit',
+                              height: 42,
+                            }}
+                          >
+                            <ListItemIcon {...providedDrag.dragHandleProps}>
+                              <DragIndicatorOutlined />
+                            </ListItemIcon>
+
+                            <ListItemText primary={formatColumnName(column)} />
+
+                            <ListItemSecondaryAction>
+                              <IconButton onClick={() => handleToggleColumn(column.attribute)}>
+                                <Close />
+                              </IconButton>
+                            </ListItemSecondaryAction>
+                          </ListItem>
+                        )}
+                      </Draggable>
+                    ))}
+                    {providedDrop.placeholder}
+                  </List>
+                )}
+              </Droppable>
+            </DragDropContext>
+          </Box>
+        </Box>
+
+        <Box sx={{ display: 'flex', marginTop: 2, justifyContent: 'flex-end' }} >
+          <Button
+            variant='outlined'
+            style={{ marginTop: '2.5px', marginLeft: '4px' }}
```

Comment:
don't know if we set such precise margin with dots... I think we recommand having only integers